autogen/oai/ollama.py

```diff
@@ -26,6 +26,7 @@
 from typing import Any, Dict, List, Tuple
 
 import ollama
+from fix_busted_json import repair_json
```

Comment:
Import "fix_busted_json" could not be resolvedPylance[reportMissingImports]
once I did `pip install fix-busted-json` it worked

Reply:
Thanks for pointing this out, I'll add to the note at the top.